docs/BuildConfiguration.md

```diff
@@ -37,6 +37,7 @@ Sections follow that describe particular settings in more depth.
 | `streets`                          | Include street input files (OSM/PBF)                                                                                                                                                                                                              | boolean        | true                    |                                                                                           |
 | `storage`                          | Configure access to data sources like GRAPH/OSM/DEM/GTFS/NETEX/ISSUE-REPORT.                                                                                                                                                                      | object         | null                    |                                                                                           |
 | `subwayAccessTime`                 | Minutes necessary to reach stops served by trips on routes of `route_type=1` (subway) from the street                                                                                                                                             | double         | 2.0                     | units: minutes                                                                            |
+| `timeZone`                         | Time zone for the graph. This is used to store the timetables in the transit model, and to interpret times in incoming requests.                                                                                                                  | string         | null                    | The value can be given either as a zone id, or an UTC offset.                             |
```

Comment:
We should mention here that if not set that the timezone is taken from the imported agencies. All agencies then need to have the same zoneId, or else OTP will not start.

Reply:
Updated